integration-test-common/build.gradle.kts

```diff
@@ -29,9 +31,15 @@ dependencies {
   testImplementation(project(":api"))
   testImplementation(project(":clients:client-java"))
   testImplementation(project(":common"))
-  testImplementation(project(":core"))
-  testImplementation(project(":server"))
-  testImplementation(project(":server-common"))
+  testImplementation(project(":core")) {
+    exclude(group = "org.rocksdb", module = "rocksdbjni")
```

Comment:
any reason exclude `rocksdb`? since this is used for test only.

Reply:
The `rocksdb-xxx.jar` have 47MB size, let Shadow jar package very big, and currently we doesn't support rocksdb storage. So I exclude it.